### PR TITLE
oracle/neofs: upgrade to SDK RC11

### PR DESCRIPTION
**Pull request by roman-khimov**
It requires explicit signers, and we have a problem with the old code:

  2023-09-12T18:42:00.063Z        WARN        failed to perform oracle request        {"url": "neofs:FMDZvqUCqcduZa8HD6wJNsHWrJ6sqkgBveGuYuL38pvH/5DCg4wUgWuWN3zsF4P4HdAzY2iKvXcrZ8QBLYGd1D1g2", "error": "failed to create client: incorrect signer: expected ECDSA_DETERMINISTIC_SHA256 scheme"}

### CODE REVIEW

File: pkg/services/oracle/neofs/neofs.go [hunk 1]
```diff
@@ -54,7 +54,6 @@ func Get(ctx context.Context, priv *keys.PrivateKey, u *url.URL, addr string) (i
 		prmi client.PrmInit
 		c    *client.Client
 	)
-	prmi.SetDefaultSigner(neofsecdsa.Signer(priv.PrivateKey))
 	c, err = client.New(prmi)
```
Comment (AnnaShaleva) [inline pkg/services/oracle/neofs/neofs.go:57]:
```suggestion
	c, err := client.New(client.PrmInit{})
```
and remove this `var` decl above then?

File: pkg/services/oracle/neofs/neofs.go [hunk 2]
```diff
@@ -127,11 +127,16 @@ func parseNeoFSURL(u *url.URL) (*oid.Address, []string, error) {
 }
 
 func getPayload(ctx context.Context, s user.Signer, c *client.Client, addr *oid.Address) (io.ReadCloser, error) {
+	var iorc io.ReadCloser
 	_, rc, err := c.ObjectGetInit(ctx, addr.Container(), addr.Object(), s, client.PrmObjectGet{})
-	return rc, err
+	if rc != nil {
+		iorc = rc
+	}
+	return iorc, err
```
Comment (AnnaShaleva) [inline pkg/services/oracle/neofs/neofs.go:135]: Wow, this one deserves the `bug` label.

**Review by AnnaShaleva — APPROVED** [review]

File: pkg/services/oracle/neofs/neofs.go (hunk 1 above)
Comment (roman-khimov) [reply pkg/services/oracle/neofs/neofs.go]: So be it.

File: pkg/services/oracle/neofs/neofs.go (hunk 2 above)
Comment (roman-khimov) [reply pkg/services/oracle/neofs/neofs.go]: ok

**Pull request merged**